code/modules/events/wizard/magicarp.dm

```diff
@@ -39,18 +39,23 @@
 	var/allowed_projectile_types = list(/obj/item/projectile/magic/change, /obj/item/projectile/magic/animate, /obj/item/projectile/magic/resurrection,
 	/obj/item/projectile/magic/death, /obj/item/projectile/magic/teleport, /obj/item/projectile/magic/door, /obj/item/projectile/magic/aoe/fireball,
 	/obj/item/projectile/magic/spellblade, /obj/item/projectile/magic/arcane_barrage)
+	var/chaos = FALSE
 	
 /mob/living/simple_animal/hostile/carp/ranged/Initialize()
 	projectiletype = pick(allowed_projectile_types)
+	if(!chaos && initial(name) == name)
+		name = "[initial(projectiletype.name)] [name]"
```

Comment:
Likewise, refactor this into a proc which is empty on `/chaos` and remove the var